src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
```

Comment:
` &:not(:nth-child(4n)) {
    margin-right: 48px;
  }`

Reply:
you can do something like this. i've the same review from mentors. i think it can work

Reply:
Done! Thx!